tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCldrFactory.java

```diff
@@ -138,8 +137,8 @@ private String getUncontainedPath(CLDRFile subset, CLDRFile superset) {
     private String differentPathValue(CLDRFile a, CLDRFile b) {
         int debugCount = 0;
         Set<String> paths = new TreeSet<>();
-        CollectionUtilities.addAll(a.iterator(), paths);
-        CollectionUtilities.addAll(b.iterator(), paths);
+        a.forEach(paths::add);
```

Comment:
most changes are just this sort of thing.
occasionally you'll see "forEachRemaining()" which is what you have to do if there's no iterable available, only an iterator.